### ISSUE TITLE
1min per page very slow

### ISSUE THREAD

**Issue by sklive** [neosun100/DeepSeek-OCR-WebUI#21]
📸 临时文件已保存: /tmp/tmpo38kezsl.png
📐 图片尺寸: 1191x1684
💬 提示词: <image>
<|grounding|>Convert the document to markdown....
🚀 开始推理...
/usr/local/lib/python3.12/dist-packages/transformers/generation/configuration_utils.py:590: UserWarning: `do_sample` is set to `False`. However, `temperature` is set to `0.0` -- this flag is only used in sample-based generation modes. You should set `do_sample=True` or unset `temperature`.
  warnings.warn(
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:None for open-end generation.
The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
The `seen_tokens` attribute is deprecated and will be removed in v4.41. Use the `cache_position` model input instead.
`get_max_cache()` is deprecated for all Cache classes. Use `get_max_cache_shape()` instead. Calling `get_max_cache()` will raise error from v4.48
=====================
BASE:  torch.Size([1, 256, 1280])
PATCHES:  torch.Size([6, 100, 1280])
=====================
The attention layers in this model are transitioning from computing the RoPE embeddings internally through `position_ids` (2D tensor with the indexes of the tokens), to using externally computed `position_embeddings` (Tuple of tensors, containing cos and sin). In v4.46 `position_ids` will be removed and `position_embeddings` will be mandatory.
✅ 推理完成，结果类型: <class 'str'>
📝 结果长度: 3441 字符
📦 找到 6 个边界框
🗑️ 临时文件已删除: /tmp/tmpo38kezsl.png
🗑️ 输出目录已清理: /tmp/ocr_8keczp6m
INFO:     202.76.26.30:19718 - "POST /ocr HTTP/1.1" 200 OK
INFO:     127.0.0.1:43580 - "GET /health HTTP/1.1" 200 OK
📸 临时文件已保存: /tmp/tmpwvntletc.png
📐 图片尺寸: 1191x1684
💬 提示词: <image>
<|grounding|>Convert the document to markdown....
🚀 开始推理...
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:None for open-end generation.
=====================
BASE:  torch.Size([1, 256, 1280])
PATCHES:  torch.Size([6, 100, 1280])
=====================
✅ 推理完成，结果类型: <class 'str'>
📝 结果长度: 3578 字符
📦 找到 4 个边界框
🗑️ 临时文件已删除: /tmp/tmpwvntletc.png
🗑️ 输出目录已清理: /tmp/ocr_thiub7gt
INFO:     202.76.26.30:19718 - "POST /ocr HTTP/1.1" 200 OK
INFO:     127.0.0.1:58556 - "GET /health HTTP/1.1" 200 OK
📸 临时文件已保存: /tmp/tmpx54z32cd.png
📐 图片尺寸: 1191x1684
💬 提示词: <image>
<|grounding|>Convert the document to markdown....
🚀 开始推理...
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:None for open-end generation.
=====================
BASE:  torch.Size([1, 256, 1280])
PATCHES:  torch.Size([6, 100, 1280])

nvidia-smi
Wed Nov 12 11:59:01 2025
+-----------------------------------------------------------------------------------------+
| NVIDIA-SMI 570.133.07             Driver Version: 570.133.07     CUDA Version: 12.8     |
|-----------------------------------------+------------------------+----------------------+
| GPU  Name                 Persistence-M | Bus-Id          Disp.A | Volatile Uncorr. ECC |
| Fan  Temp   Perf          Pwr:Usage/Cap |           Memory-Usage | GPU-Util  Compute M. |
|                                         |                        |               MIG M. |
|=========================================+========================+======================|
|   0  NVIDIA A40                     On  |   00000000:1B:00.0 Off |                    0 |
|  0%   49C    P0             90W /  300W |    9242MiB /  46068MiB |     12%      Default |
|                                         |                        |                  N/A |
+-----------------------------------------+------------------------+----------------------+
|   1  NVIDIA A40                     On  |   00000000:88:00.0 Off |                    0 |
|  0%   43C    P0             75W /  300W |     269MiB /  46068MiB |      0%      Default |
|                                         |                        |                  N/A |
+-----------------------------------------+------------------------+----------------------+

+-----------------------------------------------------------------------------------------+
| Processes:                                                                              |
|  GPU   GI   CI              PID   Type   Process name                        GPU Memory |
|        ID   ID                                                               Usage      |
|=========================================================================================|
|    0   N/A  N/A          924564      C   ...-default/llama-box-rpc-server        260MiB |
|    0   N/A  N/A         1585141      C   python                                 8968MiB |
|    1   N/A  N/A          924567      C   ...-default/llama-box-rpc-server        260MiB |
+-----------------------------------------------------------------------------------------+
